main.go

```diff
@@ -1,13 +1,21 @@
 package main
 
 import (
+	"errors"
 	"os"
 
 	"github.com/datreeio/datree/cmd"
+	"github.com/datreeio/datree/cmd/test"
 )
 
+const DEFAULT_ERR_EXIT_CODE = 1
```

Comment:
Maybe we can change the new error(ViolationsFoundError) into struct so it will contain its own exit code.